RIAssigner/data/PandasData.py

```diff
@@ -20,7 +20,9 @@ def read(self, filename: str):
         self._sort_by_rt()
 
     def write(self, filename: str):
-        self._data.to_csv(filename, index=False)
+        assert filename.endswith((".csv", ".tsv")), "File extention must be 'csv' or 'tsv'."
+        separator = define_separator(filename)
+        self._data.to_csv(filename, index=False, sep=separator)
```

Comment:
I know the name is somewhat self explanatory, but giving a hint for supported file formats in the comment would be useful.